python/internetarchive/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                internetarchive
+version             4.0.1
+revision            0
+
+categories-append   net
+license             AGPL-3
+maintainers         nomaintainer
+
+description         A Python interface to archive.org.
+long_description    {*}${description}
+
+homepage            https://github.com/jjjake/internetarchive
+
+checksums           rmd160  1a0a590e7b888c0e52c537c8f0178edcf46832f5 \
+                    sha256  8e897ee1c054edff40744e65e16d697dbfae26b0645a5fc7762cbaf88aff87b7 \
+                    size    104943
+
+python.versions     38 39 310 311 312
```

Comment:
this is not doing anything when the port name is not "py-<portname>". You should use "python.default_version in all other cases as you do correctly the line below. Please remove "python.versions"